### PR TITLE
Engegreg patch 2 readme

### PR DESCRIPTION
**Pull request by engegreg**
## Description

I adjusted the contributors so that they would appear on separate lines, since they were previously bunched up, along with removing the plotly typo, adding the docker specs and information.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

https://user-images.githubusercontent.com/44015442/139501923-7d0cefe8-457e-4105-8efb-d1b8fff45bd4.mp4

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by dagtag — APPROVED** [review]
Changes look great.  Love the linear form of the contributor names.  I think Youssef updated that python link to actually work, but I think it's ok in the end to remove it since it's not an actual license.  Clear and concise video, perfect Greg!

**Review by prockwood — COMMENTED** [review]
This all looks good, much cleaner! There are a couple of pull requests in the hopper with people adding their names: 

[Youssef Al-Yakoob](https://github.com/yalyakoob)
[Mark Porath](https://github.com/m-rath)
[Guy Altman](https://github.com/galtman5)
[Jake Harris](https://github.com/theawesomejaik)

I don't want to deprive anybody of their pull request, but we're gonna get conflicts if all of them are merged. Maybe we could just add the names to this request to simplify things?

**Comment by joshua-araujo** [comment]
Much cleaner and easier to read with this format.  Good job!

**Review by Daos21912 — APPROVED** [review]
Thanks for adding that bit for docker at the end. I think I'm going to need it.

File: README.md [hunk 1]
```diff
@@ -12,20 +12,25 @@ You can find the deployed project at [https://a.humanrightsfirstdocdb.dev/](http
...
+[Peter Rockwood](https://github.com/prockwood)  
+
+[Joshua Aurajo](https://github.com/joshua-aurajo)  
 
 
 <br>
 <br>
 
-![MIT](https://img.shields.io/packagist/l/doctrine/orm.svg)
-![Python](https://docs.python.org/3/)
```
Comment (galtman5) [inline README.md:27]: You are right in python doesn't require a license, but I still think keeping the python box there is beneficial as we can specify which python version at an eyes glance.

**Review by galtman5 — APPROVED** [review]
Everything looks good. I made a suggestion of keeping the python box but besides that, good work.

File: README.md (hunk 1 above)
Comment (engegreg) [reply README.md:27]: That is fair, I re added it at the bottom of the license/style section.

**Comment by engegreg** [comment]
> This all looks good, much cleaner! There are a couple of pull requests in the hopper with people adding their names:
> 
> [Youssef Al-Yakoob](https://github.com/yalyakoob) [Mark Porath](https://github.com/m-rath) [Guy Altman](https://github.com/galtman5) [Jake Harris](https://github.com/theawesomejaik)
> 
> I don't want to deprive anybody of their pull request, but we're gonna get conflicts if all of them are merged. Maybe we could just add the names to this request to simplify things?

Just got all the team members added, thanks for hyper linking everyone, made it quite a bit easier.

**Comment by collerhatguy** [comment]
A plus

**Review by rishimaru23 — APPROVED** [review]
Clean and easy to read!

**Review by collerhatguy — APPROVED** [review]
I like the cut of your jib fella

**Review by prockwood — APPROVED** [review]
Looks good Greg! Hopefully we can get this merged quickly, Mark has added some interesting research to the readme, but it would be good to get this update merged first to avoid conflicts.

**Pull request merged**